src/site/content/en/handbook/inclusion-and-accessibility/index.md

```diff
@@ -47,32 +81,103 @@ While _written language_ and _character_ are slightly less precise, they should
+
+Alt text should consider the context of the image, not just its content. For
+more information, see alt attribute.
+
+When appropriate, provide [alt text](/image-alt).
+
+Consider the following when writing alt text:
+
+*  Don't include phrases like "Image of" or "Photo of" or "Screenshot of".
+*  Include punctuation. When screen readers encounter punctuation, they pause
+   before continuing.
+*  Write short, descriptive alt text in 155 characters or less. Some screen
+   readers cut off reading alt text when longer than 155 characters.
+
+Review more [alt text best
+practices](https://developers.google.com/style/images#alt-text).
+
+### Avoid words in images
+
+Images with written words are not inclusive of those who don't speak the
+language and those with visual impairments. When possible, define what's represented in a `<figcaption>` beneath the image, instead of relying on the image to self-explain with included text.
+
+### Images of people
+
+Avoid images that may exclude certain audience members. Just like when you
+_write_ about people, remember to be inclusive when you _show_ people and the
+things they do and make. For example, avoid stock photos that show only male
+developers. As an alternative, check out [WOCINTECH Stock
+Photos](https://www.wocintechchat.com/blog/wocintechphotos).
 
 ## Create accessible code blocks
-Make sure code blocks are [simple](/handbook/quality/#keep-it-simple) and follow accessibility best practices. At a minimum:
-* Add [labels](/labels-and-text-alternatives/#label-form-elements) to form elements.
-* Include [alt text](/image-alt) for every image.
 
-In the rare case that you need to omit an accessibility detail to better convey what you're teaching,
+Make sure code blocks are [concise](/handbook/quality/#keep-it-simple) and
+follow accessibility best practices. At a minimum:
+
+*  Add [labels](/labels-and-text-alternatives/#label-form-elements) to form
+   elements.
+*  Don't use tabs to indent code; use spaces only.
+*  Wrap lines at 80 characters.
+*  Mark code blocks with the appropriate formatting (css, json, and more) to
+   style it appropriately.
+
+In the rare case that you need to omit an accessibility detail to better convey what you're teaching:
+
 1. Add a note explaining that the developer needs to include accessibility features.
 1. Link to the relevant post in the web.dev "[Accessible to all](/accessible)" collection.
 
-## Writing about people
-There are several excellent online guides for using inclusive language when writing about groups of people. The [Content Guide](https://content-guide.18f.gov/inclusive-language/) from 18F (a U.S. government office that helps other agencies improve their user experience) is a great place to start.
+## Write about people
```

Comment:
Nitpick: This reads weird to me, "Writing about people" sounds much more natural. I know this phrasing is consistent with the rest of the subheadings, but still.

Reply:
*Saying this as a non-native speaker, so I defer to your judgment.

Reply:
I think I prefer "Writing about people", the change sounds like a command "Write about people!" :D

Reply:
Yes, same sentiment here. To be fair, using active voice in titles and subheadings is exactly what we do recommend in the styleguide, but it's more relevant for things like "Use Squoosh to compress images" than this.